wandb/wandb_run.py

```diff
@@ -434,6 +434,34 @@ def project_name(self, api=None):
         api = api or self.api
         return api.settings('project') or self.auto_project_name(api) or "uncategorized"
 
+    def get_project_url(self, api=None, params=None):
+        params = params or {}
+        api = api or self.api
+        if api.api_key:
+            if api.settings('entity') is None and network:
```

Comment:
`network` isn't defined.  We should also share code between get_project_url and get_url